core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/AuditDAO.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.syncope.core.persistence.api.dao;
+
+import org.apache.syncope.core.persistence.api.dao.search.OrderByClause;
+import org.apache.syncope.core.persistence.api.dao.search.SearchCond;
+import org.apache.syncope.core.persistence.api.entity.AuditEntry;
+
+import java.util.List;
+import java.util.Set;
+
+public interface AuditDAO<T extends AuditEntry> {
```

Comment:
As Audit entries are not managed by Persistence Layer, but by `AuditManager` I would rather move all classes to Provisioning Layer, next to it.

Reply:
API work almost done. Going to move classes to the persistence module and clean up.

Reply:
Agree, let's first ensure things work fine - I am also not completely sure of my assert above, actually...

Reply:
Given https://issues.apache.org/jira/browse/SYNCOPE-1517 I would simply leave things as they are ATM, refactoring will come later by thath.